src/sentry/monitors/processing_errors/manager.py

```diff
@@ -131,10 +131,15 @@ def store_error(error: CheckinProcessingError, monitor: Monitor | None):
     pipeline = redis_client.pipeline(transaction=False)
     pipeline.zadd(error_set_key, {error.id.hex: error.checkin.ts.timestamp()})
     pipeline.set(error_key, serialized_error, ex=MONITOR_ERRORS_LIFETIME)
-    # Cap the error list to the `MAX_ERRORS_PER_SET` most recent errors
-    pipeline.zremrangebyrank(error_set_key, 0, -(MAX_ERRORS_PER_SET + 1))
     pipeline.expire(error_set_key, MONITOR_ERRORS_LIFETIME)
-    pipeline.execute()
+    pipeline.zrange(error_set_key, 0, -(MAX_ERRORS_PER_SET + 1))
+    results = pipeline.execute()[3]
```

Comment:
What's the 3rd tuple?

Reply:
The pipeline returns results for all commands. The `zrange` is in index 3

Reply:
I would almost be tempted to destructure and use underscore prefixes for the ones we don't care about, so it's more clear

Reply:
I'm not sure if that's clearer... what if we had 10 operations here, should we destructure all of them?

Although I wonder if it's safer to use `-1` and then try to keep this as the last operation...